client/app/components/group/groupDetail/groupDetail.html

```diff
@@ -25,7 +25,7 @@ <h1 flex editable-text="$ctrl.groupData.name">
     </div>
   </md-toolbar>
   <md-content layout-padding>
-    <div ng-if="$ctrl.$mdMedia('gt-sm')" editable-textarea="$ctrl.groupData.description">
+    <div ng-if="$ctrl.$mdMedia('gt-sm') && $ctrl.currentNavItem != 'description'" editable-textarea="$ctrl.groupData.description">
```

Comment:
this works if you open the URL, which is the common case I guess.
However, if I replace the default "pickups" in the URL with "description", ti still shows it multiple times...

I don't have an idea yet how to make it better though, just bluntly commenting ;)